src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_AGE = 18;
+    private static RegistrationService rs;
+
+    private User testUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        rs = new RegistrationServiceImpl();
```

Comment:
```suggestion
        registrationService = new RegistrationServiceImpl();
```
you should use informative names